docs/book/element/color.md

```diff
@@ -29,11 +29,11 @@ use Laminas\Form\Form;
 
 $form = new Form('my-form');
 $form->add([
-	'type' => Element\Color::class,
-	'name' => 'color',
-	'options' => [
-		'label' => 'Background color',
-	],
+ 'type' => Element\Color::class,
+ 'name' => 'color',
+ 'options' => [
+  'label' => 'Background color',
+ ],
```

Comment:
All code examples in the documentation must follow the [coding standard for Laminas components](https://docs.laminas.dev/laminas-coding-standard/).

Reply:
In this first try I just run `markdownlint-cli2-fix`.

Have you any idea on how to apply PHPCS on snippets inside doc?

We are doomed to fail without automation here.

Reply:
Unfortunately, some rules are not covered by Markdownlint so far.

Reply:
> Have you any idea on how to apply PHPCS on snippets inside doc?

Unfortunately not.

> We are doomed to fail without automation here.

The linting for Markdown was introduced without a full ruleset in place. I would deactivate it completely for now otherwise we can not use the extensions of PyMdown like [Superfences](https://facelessuser.github.io/pymdown-extensions/extensions/superfences/) or [Tabbed](https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/).

Reply:
I made it pass the QA, if it's ok for you I'd prefer to have it as is and disable it only when new tools are ready.